net_ip_socket.go

```diff
@@ -224,5 +248,14 @@ func parseNetIPSocketLine(fields []string) (*netIPSocketLine, error) {
 		return nil, fmt.Errorf("%s: Cannot parse inode value in %q: %w", ErrFileParse, line.Inode, err)
 	}
 
+	// drops
+	if isUDP {
+		drops, err := strconv.ParseUint(fields[12], 0, 64)
```

Comment:
What is fields[12] for non udp? Would it make sense to always set drops if fields[12] exist?

Reply:
Unfortunately, after the inode column, the file structure is different - in the `/proc/net/udp` file it is drops, but in `/proc/net/tcp` it is [retransmit timeout](https://www.kernel.org/doc/Documentation/networking/proc_net_tcp.txt).

Reply:
Ok makes sense